### PR TITLE
Update Onyx hash (LRU cache)

### PR DESCRIPTION
**Pull request by kidroca**
### Details

Update to use latest Onyx version
Related to PR: https://github.com/Expensify/react-native-onyx/pull/93

### Fixed Issues

N/A

### Tests

No specific tests. But a general tests for regressions
Init and chat switch performance should be the same or improved slightly

### QA Steps

Same as "Tests"

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

image

#### Mobile Web

Screenshot 2021-08-09 at 20 10 50

#### Desktop

Screenshot 2021-08-09 at 20 07 53

#### iOS

Screenshot 2021-08-09 at 20 06 55

#### Android

image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -36094,57 +36099,39 @@
...
-          "from": "git+https://github.com/Expensify/expensify-common.git#4016a786d66ff38ed76a191edef31ebd737a0190",
+          "version": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
+          "from": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
           "requires": {
-            "classnames": "2.3.1",
+            "classnames": "2.2.5",
             "clipboard": "2.0.4",
             "html-entities": "^1.3.1",
-            "jquery": "3.6.0",
+            "jquery": "3.3.1",
```
Comment (NikkiWines) [inline package-lock.json:36118]: Is there a reason we're downgrading to an older release of jQuery?

Comment (kidroca) [reply package-lock.json:36118]: It happened automatically and it seem to be the correct version acording to package specification
Onyx uses Expensify/expensify-common@2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c 
 - Onyx.package.json -> https://github.com/Expensify/react-native-onyx/blob/d7553b95e982ab78f6bb2064f6b0549f0ace94c2/package.json#L16
 - That hash of `expensify-common` uses jQuery 3.3.1 -> https://github.com/Expensify/expensify-common/blob/2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c

Probably someone committed incorrect package-lock state

Comment (NikkiWines) [reply package-lock.json:36118]: Ah yeah, it looks like we aren't consistently updating repos with expensify-common dependencies after making changes in expensify-common.

I guess it's fine to merge this as-is... cc: @marcaaron just for awareness. I think we need to be more diligent with the follow-up PRs that should be made after making changes to expensify-common.

**Review by NikkiWines — DISMISSED** [review]
@kidroca you've got some conflicts now sorry but overall looks good

**Comment by kidroca** [comment]
Fixed conflicts and re-tested

**Review by NikkiWines — APPROVED** [review]
👍

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-08-24.  :confetti_ball:

**Pull request merged**